backend/app/db/queries/sql/articles.sql

```diff
@@ -0,0 +1,78 @@
+-- name: create-new-article<!
+WITH author_subquery AS (
+SELECT id, username
+FROM users
+WHERE username = :author_username
+)
+INSERT INTO articles (headline, slug, description, body, is_drafted, author_id)
+VALUES (:headline, :slug, :description, :body, :is_drafted, (SELECT id FROM author_subquery))
```

Comment:
This won't work because the ArticlesRepository's `create_article` dosn't have the correct parameter for this kind of query.